<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1609</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>1571</width>
     <height>811</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="Tab1">
    <attribute name="title">
     <string>Visualization</string>
    </attribute>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>171</width>
       <height>351</height>
      </rect>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <column>
      <property name="text">
       <string>Objects</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>Category 01</string>
      </property>
      <item>
       <property name="text">
        <string>Type 01</string>
       </property>
       <item>
        <property name="text">
         <string>Object 01</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Object 02</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>Type 02</string>
       </property>
       <property name="icon">
        <iconset theme="printer">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
       <item>
        <property name="text">
         <string>Object 03</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Object 04</string>
        </property>
       </item>
      </item>
     </item>
     <item>
      <property name="text">
       <string>Category 02</string>
      </property>
     </item>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>750</width>
       <height>551</height>
      </rect>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget{
setColumnCount(4)
setColumnWidth(0, 200)
setColumnWidth(1, 200)
setColumnWidth(2, 200)
setColumnWidth(3, 200)
}</string>
     </property>
     <column>
      <property name="text">
       <string>Object</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Location</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Calibration</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quantity</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Category</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="insertDataButton">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>530</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Insert data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>470</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Verify</string>
     </property>
    </widget>
    <widget class="QToolBox" name="toolBox">
     <property name="geometry">
      <rect>
       <x>1120</x>
       <y>10</y>
       <width>127</width>
       <height>251</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>127</width>
        <height>183</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset>
        <normaloff>../../../../../Downloads/abstract-user-flat-1.png</normaloff>../../../../../Downloads/abstract-user-flat-1.png</iconset>
      </attribute>
      <attribute name="label">
       <string>User</string>
      </attribute>
      <widget class="QPushButton" name="profileButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Profil</string>
       </property>
      </widget>
      <widget class="QPushButton" name="projectButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Projects</string>
       </property>
      </widget>
      <widget class="QPushButton" name="signoutButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Sign Out</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset>
        <normaloff>../../../../../Downloads/chevron-up.svg</normaloff>../../../../../Downloads/chevron-up.svg</iconset>
      </attribute>
      <attribute name="label">
       <string/>
      </attribute>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>120</width>
         <height>351</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="reset_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>550</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QTreeWidget" name="treeWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>400</y>
       <width>171</width>
       <height>141</height>
      </rect>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <column>
      <property name="text">
       <string>Locations</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>Category 01</string>
      </property>
      <item>
       <property name="text">
        <string>Type 01</string>
       </property>
       <item>
        <property name="text">
         <string>Object 01</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Object 02</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>Type 02</string>
       </property>
       <item>
        <property name="text">
         <string>Object 03</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Object 04</string>
        </property>
       </item>
      </item>
     </item>
     <item>
      <property name="text">
       <string>Category 02</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="search">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>History and Tendency</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>1421</width>
       <height>561</height>
      </rect>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget{
setColumnCount(4)
setColumnWidth(0, 200)
setColumnWidth(1, 200)
setColumnWidth(2, 200)
setColumnWidth(3, 200)
}</string>
     </property>
     <column>
      <property name="text">
       <string>Object</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Location</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Calibration</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Action</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quantity</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Operation_date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Project</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btn_plot_act">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>81</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Tendency</string>
     </property>
    </widget>
    <widget class="QComboBox" name="Why_Box">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>0</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="Who_Box">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>0</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="What_Box">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>0</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="date_Box">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>0</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="action_box">
     <property name="geometry">
      <rect>
       <x>1050</x>
       <y>0</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="export_csv_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>121</width>
       <height>81</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Export_csv</string>
     </property>
    </widget>
    <widget class="QLabel" name="calendar_label">
     <property name="geometry">
      <rect>
       <x>1170</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>true</underline>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Calendar</string>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="calendar">
     <property name="geometry">
      <rect>
       <x>1050</x>
       <y>30</y>
       <width>421</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Tools localisation</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Orders</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>651</width>
       <height>461</height>
      </rect>
     </property>
     <property name="title">
      <string>User</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Request : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="request">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>41</y>
        <width>551</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>89</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Name : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="name">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>551</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>139</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Description : </string>
      </property>
     </widget>
     <widget class="QPushButton" name="send_btn">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>430</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="description">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>621</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="count_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>420</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0/1000</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableWidget" name="table_user">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>470</y>
       <width>650</width>
       <height>300</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Request</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>State</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Help</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
